docs/defi/rosetta/icrc_rosetta/running-rosetta.mdx

```diff
@@ -303,37 +282,3 @@ The `sync{token=...}` prefix indicates which tracked token is generating the log
 - **Network support**: Mainnet (test ledgers are also available on mainnet).
 - **Data persistence**: Mount `/data` directory as a volume for Docker deployments.
 - **Database files**: Each token uses a separate SQLite database when using multi-token mode.
-
-## Common configuration examples
```

Comment:
Given the updates done in the newer version of this page, these examples felt redundant and I've removed them accordingly.